dist/stat.py

```diff
@@ -32,6 +34,39 @@ def check_unique_description(sorted_list):
             print("ERROR: repeated stat description in - '%s'" % (i.desc))
         temp = i.desc
 
+##########################################
+# Remove trailing digits for a string.
+# Used as a lambda in sorting stats by name.
```

Comment:
I don't think we need to say how it's used currently as it may change overtime.
```suggestion
```

Reply:
Can we also have some context for why we strip strings here? 
Something like "Sometimes we have statistics like hist_100 and hist_50 where we want hist_50 to come first, but alphabetically hist_100 comes first. We rely on the developer to order these statistics correctly."